app/views/_alerts.html

```diff
@@ -1,28 +1,16 @@
-<div ng-attr-row="{{toast}}"  ng-attr-wrap="{{toast}}">
-  <div ng-repeat="(alertID, alert) in (alerts | filterCollection : filter) track by (alertID + (alert.message || alert.details))" ng-if="!alert.hidden" class="alert-wrapper animate-repeat" ng-class="{'animate-slide': animateSlide}">
-    <div class="alert word-break" ng-class="{
-      'alert-danger': alert.type === 'error',
-      'alert-warning': alert.type === 'warning',
-      'alert-success': alert.type === 'success',
-      'alert-info': !alert.type || alert.type === 'info',
-      'toast-pf': toast,
-      'mar-left-sm': toast
-    }">
-      <button ng-if="!hideCloseButton" ng-click="close(alert)" type="button" class="close">
+<div ng-attr-row="{{$ctrl.toast}}"  ng-attr-wrap="{{$ctrl.toast}}">
```

Comment:
See above comment, but I'd love to see this file moved to `app/scripts/components/alerts/alert.view.html`.  Colocation is 😄  😀